iron-overlay-container.html

```diff
@@ -0,0 +1,139 @@
+<!--
+@license
+Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="iron-overlay-manager.html">
+
+<!--
+`iron-overlay-container` stamps overlays to `document.body` in order to avoid
+any stacking context issue that would cause clipping or rendering behind the backdrop element.
+The rest of the content is distributed within `iron-overlay-container`.
+Mark the template containing the overlay with the attribute `overlay`.
+
+Example:
+
+    <iron-overlay-container>
+      <template overlay>
```

Comment:
I think we should use a class here instead of an attribute. See https://github.com/PolymerElements/paper-elements/issues/23